src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+constructor(url,header){
```

Comment:
The constructor of the PageObject class takes two parameters, 'url' and 'header', but only 'url' is used. If 'header' is not needed, consider removing it from the parameters.